README.md

```diff
@@ -1,12 +1,56 @@
+[![CharmHub Badge](https://charmhub.io/httprequest-lego-provider/badge.svg)](https://charmhub.io/httprequest-lego-provider)
+[![Publish to edge](https://github.com/canonical/httprequest-lego-provider/actions/workflows/publish_charm.yaml/badge.svg)](https://github.com/canonical/httprequest-lego-provider/actions/workflows/publish_charm.yaml)
+[![Promote charm](https://github.com/canonical/httprequest-lego-provider/actions/workflows/promote_charm.yaml/badge.svg)](https://github.com/canonical/httprequest-lego-provider/actions/workflows/promote_charm.yaml)
+[![Discourse Status](https://img.shields.io/discourse/status?server=https%3A%2F%2Fdiscourse.charmhub.io&style=flat&label=CharmHub%20Discourse)](https://discourse.charmhub.io)
+
 # HTTPRequest Lego Provider
 
-A Django application implementing [HTTPRequest Lego Provider](https://go-acme.github.io/lego/dns/httpreq/).
+A Juju charm deploying and managing [HTTP request Lego provider](https://go-acme.github.io/lego/dns/httpreq/)
```

Comment:
I think this would be better in a single sentence:
"A Juju charm deploying and managing [HTTP request Lego provider as defined by ACME](https://go-acme.github.io/lego/dns/httpreq/) to manage DNS records."
and I also think you can remove "HTTPRequest Lego Provider is a Let’s Encrypt client", as "LE" stands for Lest's Encrypt :)